### PR TITLE
fix: various small cleanups

### PR DESCRIPTION
**Pull request by Justintime50**
# Description

Various small cleanups:
- Require API key for creating a user
- Linted README
- Removed straggling hard-coded API keys in example files
- Removes duplicate standalone README example

# Testing

A/B tested to ensure the error of missing API key was thrown

# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement (fixing a typo, updating readme, renaming a variable name, etc)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ EASYPOST_TEST_API_KEY=123... EASYPOST_PROD_API_KEY=123... mvn clean test -B jaco
 
 ### Testing
 
-The test suite in this project was specifically built to produce consistent results on every run, regardless of when they run or who is running them. <s>This project uses [EasyVCR]() to record and replay HTTP requests and responses via "cassettes". When the suite is run, the HTTP requests and responses for each test function will be saved to a cassette if they do not exist already and replayed from this saved file if they do, which saves the need to make live API calls on every test run.</s>
+The test suite in this project was specifically built to produce consistent results on every run, regardless of when they run or who is running them. <s>This project uses [EasyVCR](https://github.com/EasyPost/easyvcr-java) to record and replay HTTP requests and responses via "cassettes". When the suite is run, the HTTP requests and responses for each test function will be saved to a cassette if they do not exist already and replayed from this saved file if they do, which saves the need to make live API calls on every test run.</s>
```
Comment (nwithan8) [inline README.md:113]: I mean, it doesn't use EasyVCR yet. How did this get here in the first place?

File: src/main/java/com/easypost/app/WebhookExample.java [hunk 2]
```diff
@@ -9,10 +9,10 @@
 public class WebhookExample {
 
     public static void main(final String[] args) {
-        EasyPost.apiKey = "cueqNZUb3ldeWTNX7MU3Mel8UXtaAMUi";
+        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
```
Comment (nwithan8) [inline src/main/java/com/easypost/app/WebhookExample.java:12]: Do we want to set these up to pull from the environment? They're just examples, why not just put a fake `secret_api_key` dummy text?

**Review by nwithan8 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Justintime50) [reply README.md:113]: We copied in the whole section when we started working on it months ago. It is stiked through though in the README itself. We'll be shortly adopting it

File: src/main/java/com/easypost/app/WebhookExample.java (hunk 2 above)
Comment (Justintime50) [reply src/main/java/com/easypost/app/WebhookExample.java:12]: Best practice to pull in creds not from directly in a script. Sensitive info like this should never be hard coded anyway.

**Review by jchen293 — APPROVED** [review]

**Pull request merged**